### ISSUE TITLE
Tensorboard is not able to find the event data 

### ISSUE THREAD

**Issue by sairaghava7** [tensorflow/tensorboard#5382]
I'm trying to visualize my high dimension data using the Tensorflow embedding projector tool and I'm not able to solve this error. It's been so many hours that I have been trying all other options but nothing worked. My event files are located in this directory 'C:\Users\sethuri\Documents\VectorSpaceModels\NEW\logs\emb' and I've tried to invoke the tensorboard but has been unsuccessful all this time. Error is **No dashboards are active for the current data set**. Any help is appreciated

![tferror2](https://user-images.githubusercontent.com/30205127/137845835-98ccc417-0836-4b66-9ccb-a81ab95a7fd5.PNG)

**Comment by UsharaniPagadala** [comment]
@sairaghava7 
Could you please refer this [comment](https://stackoverflow.com/questions/47113472/tensorboard-error-no-dashboards-are-active-for-current-data-set), and the similar [issue](https://github.com/tensorflow/tensorboard/issues/1306) , hope it helps.Thanks

**Comment by sairaghava7** [comment]
@UsharaniPagadala I have tried all the answers in the stackoverflow link that you have shared with me and also looked into the github issue link. Although in the Github issue the user was using ubuntu and mine was windows, I see that he is using space after logdir which is incorrect. 
As you can see from my post that there are event files located in the ./logs/emb folder and I CD'd into emb folder and invoked the tensorboard. I also tried invoking from logs folder but it didn't work. 

Also, I noticed that no matter which directory i'm invoking the tensorboard from, i see the log directory to be constant as indicated in the below image

![tferror3](https://user-images.githubusercontent.com/30205127/137976534-825c4b9a-c1e1-4c9b-9e04-9ec01514b823.PNG)
.
Please let me know if you could think of any other workaround. Thanks!

**Comment by yatbear** [comment]
Hi @sairaghava7, I don't see any events files (usually of format `events.out.tfevents.<TIMESTAMP>.<MACHINE_NAME>`) present under your `logdir`, see https://github.com/tensorflow/tensorboard#my-tensorboard-isnt-showing-any-data-whats-wrong for more details.

**Comment by sairaghava7** [comment]
> Hi @sairaghava7, I don't see any events files (usually of format `events.out.tfevents.<TIMESTAMP>.<MACHINE_NAME>`) present under your `logdir`, see https://github.com/tensorflow/tensorboard#my-tensorboard-isnt-showing-any-data-whats-wrong for more details.

**Comment by sairaghava7** [comment]
Yes - something is happening over here. For a day I just stopped what I was working with and ran the code from TensorFlow embedding projector code samples - https://www.tensorflow.org/tensorboard/tensorboard_projector_plugin. I didn't make any changes to the code except from changing `log_dir='/logs/imdb-example/'` to log_dir='./logs/imdb-example/' since I'm working on windows machine. The code got executed successfully and the projector files got generated.

![image](https://user-images.githubusercontent.com/30205127/140092882-24c79416-e4af-4781-9574-4cb746423fac.png)

I invoked tensorboard from my jupyter instance %tensorboard --logdir=./logs/imdb-example/ but still seeing the same error.
![image](https://user-images.githubusercontent.com/30205127/140093127-8cfb4ca4-9f66-4805-b996-ea33cc2a0424.png)

I have uninstalled and reinstalled anaconda before experimenting with the code from tensorflow website and i see no change in the output. I'm neither getting output on my data nor on the code from embedding projector tutorial from tensorflow
